### PR TITLE
Task-44074: Add a warning message about links on share activity drawer

### PR DESCRIPTION
**Pull request by hassenelkamel-exo**
This PR will add a warning message about links on share activity drawer

### CODE REVIEW

File: webapp/portlet/src/main/webapp/share-activity-drawer/components/ShareActivityDrawer.vue [hunk 1]
```diff
@@ -27,6 +27,10 @@
                 class="ml-2 ignore-vuetify-classes activityShareDescription">
               </textarea>
             </div>
+            <div class="d-flex flex-row mt-4 ml-2">
+              <v-icon size="20px" color="#f8b441">warning</v-icon>
```
Comment (azayati) [inline webapp/portlet/src/main/webapp/share-activity-drawer/components/ShareActivityDrawer.vue]: - color and size should not be added directly, use css helpers instead
- the warning message should be added only if the description contains a link

**Review by azayati — CHANGES_REQUESTED** [review]
See review

File: webapp/portlet/src/main/webapp/share-activity-drawer/components/ShareActivityDrawer.vue (hunk 1 above)
Comment (hassenelkamel-exo) [reply webapp/portlet/src/main/webapp/share-activity-drawer/components/ShareActivityDrawer.vue]: -done

File: webapp/portlet/src/main/webapp/share-activity-drawer/components/ShareActivityDrawer.vue [hunk 2]
```diff
@@ -54,6 +58,13 @@ export default {
     shareDisabled() {
       return !this.spaces || this.spaces.filter(part => part !== '').length === 0;
     },
+    checkUrl() {
```
Comment (azayati) [inline webapp/portlet/src/main/webapp/share-activity-drawer/components/ShareActivityDrawer.vue]:
```suggestion
    descriptionHasLink() {
```

File: webapp/portlet/src/main/webapp/share-activity-drawer/components/ShareActivityDrawer.vue [hunk 3]
```diff
@@ -54,6 +58,13 @@ export default {
...
+      if (this.description) {
```
Comment (azayati) [inline webapp/portlet/src/main/webapp/share-activity-drawer/components/ShareActivityDrawer.vue]:
```suggestion
      return this.description && this.description.match(REGEX);
```
the description should contain at least one link having one the below pattern:
https://www.XXXX.YYYY
http://www.XXXX.YYYY
www.XXXX.YYYY
https://XXXX.YYYY
http://XXXX.YYYY

**Review by azayati — CHANGES_REQUESTED** [review]
See review

**Review by azayati — DISMISSED** [review]
LGTM after QA validation

**Review by azayati — DISMISSED** [review]
LGTM after QA validation

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**